dspace-spring-rest/src/main/java/org/dspace/app/rest/model/SubmissionFormFieldRest.java

```diff
@@ -85,6 +87,14 @@ public void setHints(String hints) {
         this.hints = hints;
     }
 
+    public String getStyle() {
```

Comment:
We should add Javadocs to these new methods.  It's not entirely clear what a "Style" is.... is any CSS style valid here?